lib/plugins/digging.js

```diff
@@ -110,20 +121,30 @@ function inject (bot) {
       location: block.position,
       face: diggingFace // default face is 1 (top)
     })
-    const waitTime = bot.digTime(block)
-    waitTimeout = setTimeout(finishDigging, waitTime)
+    if (waitInterval) { clearInterval(waitInterval) }
+    waitInterval = setInterval(() => {
+      bot.digPercentage += (50 / bot.digTime(block))
+      if (bot.digPercentage >= 1.0) {
+        finishDigging()
+      }
+    }, 50)
     bot.targetDigBlock = block
     bot.swingArm()
 
     swingInterval = setInterval(() => {
-      bot.swingArm()
-    }, 350)
+      if (bot.targetDigBlock) {
+        bot.swingArm()
+      } else {
+        clearInterval(swingInterval)
+      }
+    }, 250)
```

Comment:
why the change from 350 to 250 ?

Reply:
@Vakore can you change this back or give w reason to why you changed the interval?

Reply:
I changed the interval to fix the arm swing bug